src/server/supplements/debugger.ts

```diff
@@ -47,21 +48,33 @@ export class Debugger extends EventEmitter implements InstrumentationListener {
     return (context as any)[symbol] as Debugger | undefined;
   }
 
+  async setMuted(muted: boolean) {
+    this._muted = muted;
+  }
+
   async onBeforeCall(sdkObject: SdkObject, metadata: CallMetadata): Promise<void> {
-    if (shouldPauseOnCall(sdkObject, metadata) || (this._pauseOnNextStatement && shouldPauseOnStep(sdkObject, metadata)))
+    if (this._muted)
+      return;
+    if (shouldPauseOnCall(sdkObject, metadata) || (this._pauseOnNextStatement && shouldPauseOnNonInputStep(sdkObject, metadata)))
       await this.pause(sdkObject, metadata);
   }
 
   async onBeforeInputAction(sdkObject: SdkObject, metadata: CallMetadata): Promise<void> {
+    if (this._muted)
+      return;
     if (this._enabled && this._pauseOnNextStatement)
       await this.pause(sdkObject, metadata);
   }
 
   async onCallLog(logName: string, message: string, sdkObject: SdkObject, metadata: CallMetadata): Promise<void> {
+    if (this._muted)
```

Comment:
I think this one should be removed - if I started with DEBUG=pw:api, let me see the logs no matter what :)